main/CMakeLists.txt

```diff
@@ -118,6 +123,13 @@ if (WIN32)
   	target_link_libraries(minifiexe semaphore)
 endif()
 
+if (NOT USE_SHARED_LIBS)
```

Comment:
Doesn't really matter, just readibility: why not using an AND here instead of nested conditions?